.travis.yml

```diff
@@ -27,24 +39,28 @@ before_install:
 install:
   # Upgrade pip and setuptools. Mock has issues with the default version of
   # setuptools
-  - |
-    pip install --upgrade pip
-    pip install --upgrade setuptools
-    pip install $NUMPY
-    pip install $MPL
-    pip install -r requirements.txt
+  - pip install --upgrade pip
+  - pip install --upgrade setuptools
+
+  - pip install $NUMPY
+  - pip install $MPL
```

Comment:
Spelling APT package names correctly is very important.  Installing via APT geos, freetype and libpng causes typical builds to be 2-3 minutes (per instance).  Compiling geos builds take about 6-7 minutes.  Where build times were taking 8-10 minutes.